packages/mobile/patches/react-native+0.66.3.patch

```diff
@@ -1,13 +0,0 @@
-diff --git a/node_modules/react-native/Libraries/Animated/AnimatedEvent.js b/node_modules/react-native/Libraries/Animated/AnimatedEvent.js
-index b6de725..dfb9777 100644
---- a/node_modules/react-native/Libraries/Animated/AnimatedEvent.js
-+++ b/node_modules/react-native/Libraries/Animated/AnimatedEvent.js
-@@ -227,7 +227,7 @@ class AnimatedEvent {
-   }
- 
-   _callListeners(...args: any) {
--    this._listeners.forEach(listener => listener(...args));
-+    this._listeners.forEach(listener => listener?.(...args));
```

Comment:
Was this fixed in react-native 0.68? So we don't need the patch anymore?

Reply:
yeah i wasnt seeing this issue happen in my local testing anymore